js/views/PageNav.js

```diff
@@ -0,0 +1,67 @@
+import electron from 'electron';
+import { View } from 'backbone';
+import loadTemplate from '../utils/loadTemplate';
+import app from '../app';
+
+const remote = electron.remote;
+
+export default class PageNav extends View {
```

Comment:
As long as it's a hard rule that the nav bar is never covered, I think that's fine. 

But that raises questions about situations where the navbar should not be functional, such as when the server can't be contacted, or during onboarding.